EventHubs/tests/test_eventhub_metrics.py

```diff
@@ -1,48 +1,111 @@
+import sys
 import unittest
-import datetime
+from io import StringIO
+from datetime import timedelta
 from baseeventhubtest import BaseEventHubTest
-
-
+from azure.monitor.query import LogsQueryClient, LogsQueryStatus
 class TestEventHubMetrics(BaseEventHubTest):
 
     def setUp(self):
-        super(TestEventHubMetrics, self).setUp()
-        self.RESOURCE_GROUP_NAME = "TestEventHubMetrics-%s" % (
```

Comment:
why did we remove this ? we should not hard code resource group name

Reply:
revert it back to strftime why doing unnecessary int str conversions

Reply:
updated